### ISSUE TITLE
IT Support: Network issues at Dalton. 

### ISSUE THREAD

**Issue by Matthew-M-TPW** [cityofaustin/atd-data-tech#17021]
DALTON

## Device & User Information (first name and last initial, no last names)
Ali, A
## What's the problem?
[10:36 AM]  Ali
Hello Matthew, 
[10:37 AM] , Ali
This is Ali from NIO. I am reaching out to you regarding the ticket you submitted INC0385590. 
[10:38 AM] , Ali
I did send Brad but he did not respond yet. Are you the IT Support in this site? was there any troubleshooting done by the IT Support?

**Comment by Matthew-M-TPW** [comment]
Brad noticed upon entering the building that there was no internet access. After checking his phone for a dial tone and finding none, he proceeded to reset it. This has been happening to all the offices in the building randomly. 

At Dalton, I joined Brad and Ali in troubleshooting. We checked the connections and restarted several machines in an attempt to replicate the issue.

Further investigation revealed that the gate, which is network-connected, was supposed to open at 6 AM but did not. I disconnected the ethernet cable from the gate's controls, resulting in a successful reset. We suspect there may be an issue with the switch, and Ali plans to discuss this with his boss.
